rust/opendp_tooling/src/bootstrap/docstring.rs

```diff
@@ -26,6 +33,23 @@ impl BootstrapDocstring {
         output: &ReturnType,
         path: Option<(&str, &str)>,
     ) -> Result<BootstrapDocstring> {
+        // look for this attr:
+        // #[deprecated(note="please use `new_method` instead")]
+        let deprecated = attrs
+            .iter()
+            .find(|attr| {
+                attr.path.get_ident().map(ToString::to_string).as_deref() == Some("deprecated")
+            })
+            .map(|attr| {
+                // TODO: more nicely format this message
+                Result::Ok(
+                    DeprecationArguments::from_meta(&attr.parse_meta()?)?
+                        .note
+                        .unwrap_or_default(),
+                )
```

Comment:
If you're building a deprecated struct here, then something like this could work:
```suggestion
                let meta = DeprecationArguments::from_meta(&attr.parse_meta()?)?;
                Result::Ok(Deprecated {
                    version = meta.version.unwrap_or_default(),
                    note = meta.note.unwrap_or_default()
                })
```